### ISSUE TITLE
Query Invoice  Tests  Not  Supported - 29/10/2018 - 5004

### ISSUE THREAD

**Issue by SME-Issues** [SME-Issues/issues#25]
**Query Invoice  Tests  Not  Supported**
- Total: 18 (100%)
- Not understood or error: 5 (28%)
- Subtotal: 13 (72%)
- Understood and passed: 7 (54%)
- Understood but failed: 6 (46%)

**Comment by SME-Issues** [comment]
### Understood and failed


### Fail - Understanding: Full - NotYetSupported
|Expressions|Notes|
|---|---|
| _Do people owe me more than I owe at the moment?_ |Expected ctx&hairsp;.SingleContext&hairsp;.UnsupportedType to be Comparison, but found None.|
| _are my bills settled for december_ |Failed because dependent on #3049&hairsp;. expected intent to be `query_invoices` but found `query_payment`|
| _are my invoices settled for this month_ |Failed because dependent on #3049&hairsp;. Expected ctx&hairsp;.SingleContext&hairsp;.DueDateQueryEnd to be <2018-10-31>, but found <9999-12-31>.|
| _are all my invoices settled for oct_ |Failed because dependent on #3049&hairsp;. Expected ctx&hairsp;.SingleContext&hairsp;.DueDateQueryStart to be <2017-10-01>, but found <0001-01-01 00:00:00&hairsp;.000>.|
| _what will my cash position be after I pay all my invoices._ |NotYetSupported - complex query|
| _do you know if alpha ltd and beta ltd has paid yet_ |NotYetSupported - multiple accounts in the same query&hairsp;. Chunk count should be 1, though.|

**Comment by SME-Issues** [comment]
### Failed as not understood


### Fail - Understanding: None - NotYetSupported
|Expressions|Notes|
|---|---|
| _When I pay invoice 14122 how much cash is there?_ |Expression not understood: '14122'|
| _Can you tell me if this month’s invoice from ABC is bigger than last month’s?_ |Failed because queries of this type are out of scope&hairsp;. Expression not understood: 'can you tell me if this month's invoice from abc is bigger than last month's'|
| _If I pay invoice 14124 how much will I have left?_ |Failed because querying invoices by number is out of scope&hairsp;. NotYetSupported|
| _Is there more than £70k owed?_ |NotYetSupported|

### Fail - Understanding: None - TestFailedException
|Expressions|Notes|
|---|---|
| _how many times has Sam's Deli been late settling their account in the last year_ |System&hairsp;.InvalidOperationException: Unsupported intent: 'multi_query'   at PulseAPI&hairsp;.Processors&hairsp;.InputProcessor&hairsp;.ProcessChunks(IDialogueState previousDialogueState, IDialogueState newDialogueState, AiResp…|
